### PR TITLE
Adapt w.r.t. coq/coq#14684

### PR DESCRIPTION
**Pull request by ppedrot**
Don't merge yet.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -115,6 +115,12 @@ ML_DESCRIPTION := "Coq v8.14"
 OTHERFLAGS += -w "-deprecated-appcontext -notation-overridden"
 ML4_OR_MLG := mlg
 else
+ifneq (,$(filter 8.15%,$(COQ_VERSION)))
+EXPECTED_EXT:=.v815
+ML_DESCRIPTION := "Coq v8.15"
+OTHERFLAGS += -w "-deprecated-appcontext -notation-overridden"
```
Comment (SkySkimmer) [inline Makefile]: appcontext hasn't existed for a while

Comment (ppedrot) [reply Makefile]: I cargo-culted the code for the 8.14 version, don't slap my hands.

**Comment by ppedrot** [comment]
This needs to be merged now.

**Comment by ppedrot** [comment]
Ping.

**Comment by JasonGross** [comment]
Sorry for the delay, I wanted to make sure the repo was synced with our private one before merging

**Comment by JasonGross** [comment]
Merged now, thanks!

**Pull request merged**